index.html

```diff
@@ -165,8 +165,24 @@ <h2>Multi-Post Stories</h2>
                 </div>
             </section>    
         </div>
+        <section id="container2">
+            <h1>Contact me </h1>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+            <form action="https://formspree.io/f/mdovbonp" method="post">
+                <label for="fullname">
```

Comment:
[OPTIONAL]
- [ ] The `label` attribute here doesn't have any semantic meaning, you may consider deleting it. and also when using it make sure you use it correctly. like so:
```html
<label for="some_text">Label_name</label>
```

Reply:
@christianbiring1 Accept my deepest thanks for your assistance and I will make sure that I deleted it.
The reason why I did it like that was, we must stick to the design as much as possible (e.g., font, colors, images, **text**, margins) using the Figma Template in the curriculum, that is why I didn't write anything inside the label attribute. 😃 
Once again, my gratitude knows no bounds

Reply:
Yeah exactly, since the design doesn't have any label, that's the reason why I have suggested deleting it.